.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) Riverside Valley Corporation
```

Comment:
As above.

Reply:
As I mentioned in https://github.com/nerocui/JitHubV2/pull/3#discussion_r1193697269